src/messageArea.js

```diff
@@ -4,7 +4,6 @@ var UI = {
   authn: require('./authn/authn'),
   icons: require('./iconBase'),
   ns: require('./ns'),
-  pad: require('./'),
```

Comment:
Is this change necessary in this PR?

Reply:
Not strictly but it is related and life is too short to separate them